### PR TITLE
T 16562 post hook within all models config for tblproperties

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:

This PR marks the end of the alter_table_properties macro 🎉, I replaced all tblproperties attributed to spells within all the existing models that need to be exposed on the Dune Data explorer 

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by soispoke** [comment]
tagging active wizards for visibility @hildobby @bh2smith @catherine-2 @0xRobin @RplusT @dsalv

File: models/cow_protocol/ethereum/cow_protocol_ethereum_batches.sql [hunk 1]
```diff
@@ -4,7 +4,11 @@
         unique_key = ['tx_hash'],
         on_schema_change='sync_all_columns',
         file_format ='delta',
-        incremental_strategy='merge'
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "cow_protocol",
+                                    \'["bh2smith", "gentrexha"]\') }}'
```
Comment (bh2smith) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_batches.sql:11]: Looks good here and all others cow_protocol.

**Review by bh2smith — APPROVED** [review]
Seems legit. The macro file was deleted and all existing SQL file with spells defined have an added  hook.

**Comment by 0xRobin** [comment]
> tagging active wizards for visibility @hildobby @bh2smith @catherine-2 @0xRobin @RplusT @dsalv

Awesome, less files to change for adding a new model!
Please also update these to reflect the changes. :pray: 
https://github.com/duneanalytics/spellbook/blob/b8b51236e9acb79ab08a07384d81e2cd7399fc6b/pull_request_template.md?plain=1#L14
and
https://github.com/duneanalytics/spellbook/blob/b8b51236e9acb79ab08a07384d81e2cd7399fc6b/CONTRIBUTING.md?plain=1#L99-L114

File: models/labels/cex/labels_cex_ethereum.sql [hunk 2]
```diff
@@ -1,496 +1,12 @@
...
+                                    "sector",
+                                    "labels",
+                                    \'["hildobby"]\') }}')}}
 
 SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
 FROM (VALUES
     -- Binance, Source: https://etherscan.io/accounts/label/binance
     (array('ethereum'),'0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be', 'Binance 1', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
     , (array('ethereum'), '0xd551234ae421e3bcba99a0da6d736074f22192ff', 'Binance 2', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
-    , (array('ethereum'), '0x564286362092d8e7936f0549571a803b203aaced', 'Binance 3', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
```
Comment (aalan3) [inline models/labels/cex/labels_cex_ethereum.sql:8]: Why drop this?

Comment (soispoke) [reply models/labels/cex/labels_cex_ethereum.sql:8]: @aalan3 it shouldn't have been, thanks for spotting this !

File: pull_request_template.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ General checks:
 * [ ] each sql file is a select statement and has only one view, table or function defined  
 * [ ] column names are `lowercase_snake_cased`
 * [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
-* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
+* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, nam and contributors)
```
Comment (jeff-dude) [inline pull_request_template.md:14]: minor typo here on 'name'

Comment (soispoke) [reply pull_request_template.md]: thanks ! fixed ✔️

**Review by jeff-dude — APPROVED** [review]
lfg

**Pull request merged**